enumfields/fields.py

```diff
@@ -11,6 +11,9 @@
 from .forms import EnumChoiceField
 
 
+INITIAL_FIELD_TMP = '_initial_{}'
```

Comment:
will be cool to have function on EnumFieldValidationMixin to get cache value name. The same logic is used on FK field, it is method called **get_cache_name**. Maybe we should use get_initial_cache_name

Reply:
good idea, changed.